lntest/itest/lnd_route_blinding.go

```diff
@@ -0,0 +1,291 @@
+package itest
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+
+	"github.com/btcsuite/btcd/btcec/v2"
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/lightningnetwork/lnd/chainreg"
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lnrpc/routerrpc"
+	"github.com/lightningnetwork/lnd/lntemp"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// testQueryBlindedRoutes tests querying routes to blinded routes.
+func testQueryBlindedRoutes(ht *lntemp.HarnessTest) {
+	var (
+		ctxb = context.Background()
+
+		// Convenience aliases.
+		alice = ht.Alice
+		bob   = ht.Bob
+	)
+
+	// Setup a two hop channel network: Alice -- Bob -- Carol.
```

Comment:
I don't see how it would _increase_ coverage of pathfinding to a blinded route to cover _beyond_ the introduction node?

Also, how would you add a receiver node when LND doesn't support receiving?

Reply:
Yes, good point, right now it doesn't make sense :+1: